meta/implementation.md

```diff
@@ -230,3 +230,24 @@ We have found some libraries that can be used for an implementation:
 - Julia: [Statistics.quantile](https://docs.julialang.org/en/v1/stdlib/Statistics/#Statistics.quantile!), type 7 is the default.
 - Python: [numpy](https://numpy.org/doc/stable/reference/generated/numpy.quantile.html), [pandas](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.quantile.html), [xarray](http://xarray.pydata.org/en/stable/generated/xarray.DataArray.quantile.html) - type 7 (called 'linear' for the interpolation parameter) is the default for all of them.
 - R: [quantile](https://stat.ethz.ch/R-manual/R-patched/library/stats/html/quantile.html) - type 7 is the default.
+
+## STAC support
+
+The subtype `stac` is an abstract type that refers to a STAC resource of any type (Catalog, Collection, or Item).
+It can refer to:
+- static STAC resources, e.g. hosted on cloud storage
+- "dynamic" STAC resources made available via a STAC API
```

Comment:
Just for style alignment: should we use the quotation marks also for `static` or remove them from `dynamic`?